docs/library/pyb.Timer.rst

```diff
@@ -18,6 +18,13 @@ Example usage to toggle an LED at a fixed frequency::
     tim.init(freq=2)                # trigger at 2Hz
     tim.callback(lambda t:pyb.LED(1).toggle())
 
+Example using named function for the callback::
+
+    def tick(timer):                # we will receive the timer object when being called
+        print(timer.counter())      # show current timer's counter value
+    tim = pyb.Timer(4, freq = 1)    # create a timer object using timer 4 - trigger at 1Hz
```

Comment:
PEP8 requires no spaces around "=" with keyword arguments. And as these is supposed to give example to novices to follow, it's particularly not good idea to use it like that here.